src/modules/esp32/AudioModule.cpp

```diff
@@ -1,12 +1,11 @@
-
 #include "configuration.h"
 #if defined(ARCH_ESP32) && defined(USE_SX1280)
 #include "AudioModule.h"
 #include "FSCommon.h"
 #include "MeshService.h"
-#include "NodeDB.h"
 #include "RTC.h"
 #include "Router.h"
+#include "nodeDB.h"
```

Comment:
That include is not found here... needs a capital N

Reply:
Doh! Fixed.